pages/events.js

```diff
@@ -20,6 +21,8 @@ export async function getServerSideProps(context) {
 }
 
 export default function Events({ events }) {
+
+  const [eventType, seteventType] = useState('');
```

Comment:
`useState` will need an import

Reply:
thank eddie, for letting me know I just added it